src/main/default/lwc/flowProgressIndicator/__tests__/flowProgressIndicator.test.js

```diff
@@ -0,0 +1,168 @@
+    label: 'Provide details',
+    value: 'provide details'
+  },
+  {
+    label: 'Review help',
+    value: 'review help'
+  }
+];
+
+const CUSTOM_STEPS = [
+  {
+    label: 'First step',
+    value: 'first step'
+  },
+  {
+    label: 'Second step',
+    value: 'second step'
+  }
+];
+
+const assertProgressStepEquals = (actual, expected) => {
+  expect(actual).not.toBeNull();
+  expect(actual.label).toBe(expected.label);
+  expect(actual.value).toBe(expected.value);
+};
+
+describe('c-flow-progress-indicator', () => {
+  afterEach(() => {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+  });
+
+  it('should display the progress indicator', async () => {
+    const element = createTestComponent();
+
+    await allPromisesResolution();
+    const progressIndicatorNode = element.shadowRoot.querySelector(
+      'lightning-progress-indicator'
+    );
+
+    expect(progressIndicatorNode).not.toBeNull();
+    await expect(element).toBeAccessible();
+  });
+
+  it('should display the correct default progress indicator steps', async () => {
+    const element = createTestComponent();
+
+    await allPromisesResolution();
+    const progressIndicatorNode = element.shadowRoot.querySelector(
+      'lightning-progress-indicator'
+    );
+    const stepNodes = element.shadowRoot.querySelectorAll(
+      'lightning-progress-step'
+    );
+
+    expect(progressIndicatorNode).not.toBeNull();
+    expect(stepNodes.length).toBe(DEFAULT_STEPS.length);
+    stepNodes.forEach((step, idx) =>
+      assertProgressStepEquals(step, DEFAULT_STEPS[idx])
+    );
+    await expect(element).toBeAccessible();
+  });
+
+  it('should display the correct progress indicator steps', async () => {
+    const element = createTestComponent();
+    element.steps = CUSTOM_STEPS;
+
+    await allPromisesResolution();
+    const progressIndicatorNode = element.shadowRoot.querySelector(
+      'lightning-progress-indicator'
+    );
+    const stepNodes = element.shadowRoot.querySelectorAll(
+      'lightning-progress-step'
+    );
+
+    expect(progressIndicatorNode).not.toBeNull();
+    expect(stepNodes.length).toBe(CUSTOM_STEPS.length);
+    stepNodes.forEach((step, idx) =>
+      assertProgressStepEquals(step, CUSTOM_STEPS[idx])
+    );
+    await expect(element).toBeAccessible();
+  });
+
+  it('should display an error in the current step when triggering an error', async () => {
+    const element = createTestComponent();
+
+    await allPromisesResolution();
+    const progressIndicatorNode = element.shadowRoot.querySelector(
+      'lightning-progress-indicator'
+    );
+
+    await element.triggerError(true);
+    expect(progressIndicatorNode).not.toBeNull();
+    expect(progressIndicatorNode.hasError).toBe(true);
+    await expect(element).toBeAccessible();
+  });
+
+  it('should display the correct current step', async () => {
```

Comment:
I would suggest to add two more tests to validate when `element.currentStep` is not set, and when `element.currentStep` does not match any step value.